### ISSUE TITLE
I can't get account info after connect metamask

### ISSUE THREAD

**Issue by bluedragon1120** [Shmoji/web3-react-example#1]
I cloned your source code and run it in my local.
when I check your video, it looks everything is okay.
but when I run your source, it is not working correctly.
for example, I can't get account name after connect metamask

**Comment by blakewood84** [comment]
yeah same here. To get this to work I connected a local Ginache test server to the connectors.js file. The chain ID is usually 1337. Just make sure your local chain is spun up and running

**Comment by jtibrewal09** [comment]
Can anyone please help me setup this flow. I'm new to web3 world.
